packages/trace-viewer/src/ui/workbenchLoader.tsx

```diff
@@ -191,6 +189,14 @@ export const WorkbenchLoader: React.FunctionComponent<{
         <div>3. Drop the trace from the download shelf into the page</div>
       </div>
     </div>}
+    <Dialog open={progress.done !== progress.total && progress.total !== 0} isModal={true} className='progress-dialog'>
+      <div className='progress-content'>
+        <div className='title' role='heading' aria-level={1}>Loading Playwright Trace...</div>
```

Comment:
why isn't this a h1?

Reply:
Good question. I am matching the other similar pieces of UI in Trace Viewer, like the drop target.